contentctl/objects/dashboard.py

```diff
@@ -0,0 +1,96 @@
+from typing import Any
+from pydantic import Field, Json, model_validator
+
+import pathlib
+import copy
+from jinja2 import Environment
+import json
+from contentctl.objects.security_content_object import SecurityContentObject
+from contentctl.objects.config import build
+
+DEFAULT_DASHBAORD_JINJA2_TEMPLATE = '''<dashboard version="2" theme="light">
```

Comment:
Even though this is a departure from what we have done in the past (defining the template in a separate .j2 file), I think this is quite intuitive and reduces external dependencies. Happy to hear thoughts and feedback.

Reply:
LGTM! 

Separate conversation we had about adding support for `theme="light"` vs `theme="dark"` in the yaml